lux/utils/debug_utils.py

```diff
@@ -7,6 +7,7 @@
 import re
 import subprocess
 from typing import Optional
+from packaging import version
```

Comment:
Is this an extra library dependency or something that comes by default with Python? It would be good if we can minimize library dependencies.

Reply:
https://pypi.org/project/packaging/